sonar-zap-plugin/src/main/java/org/sonar/zaproxy/parser/ReportParser.java

```diff
@@ -112,14 +115,34 @@ private AlertItem processAlertItem(SMInputCursor alertItemCursor) throws XMLStre
                 alertItem.setRiskdesc(StringUtils.trim(childCursor.collectDescendantText(false)));
             } else if ("desc".equals(nodeName)) {
                 alertItem.setDesc(StringUtils.trim(childCursor.collectDescendantText(false)));
-            } else if ("uri".equals(nodeName)) {
```

Comment:
Some of the reports might still be generated this way, the merge of the alerts is optional. [1]

[1] https://github.com/zaproxy/zap-core-help/wiki/HelpUiDialogsOptionsAlert#merge-related-alerts-in-report

Reply:
The referenced option needs to be deselected to produce the "old" format.